vassal-app/src/main/java/VASSAL/counters/Footprint.java

```diff
@@ -754,11 +755,13 @@ public void requireNoOuterRotate() {
     Decorator piece = getOuter();
```

Comment:
Would it be better to check `GameModule.getGameModule().isEditorOpen()` right at the top, before doing any other work? This way, we're traversing the trait stack when we could already know we won't report.

Reply:
Yep - fixed